### PR TITLE
Add io.github.guillaumechereau.Goxel

### PR DESCRIPTION
**Pull request by Alexander-Wilms**

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
I've created a PR that adds the appdata file upstream.

File: goxel.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (barthalion) [inline goxel.appdata.xml]: Misses releases section.

Comment (Alexander-Wilms) [reply goxel.appdata.xml]: Done

**Comment by barthalion** [comment]
bot, build io.github.guillaumechereau.Goxel

**Comment by Alexander-Wilms** [comment]
All builds succeeded: https://flathub.org/builds/#/builders/2/builds/641

File: io.github.guillaumechereau.Goxel.json [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+                "sha256": "b7d55e13e07095119e7d5f6792586dd0849c9fcdd867d49a4a5ac31f982f7326"
+            }
+        ],
+        "cleanup": [
+            "/include",
+            "/lib/pkgconfig"
+        ]
+    },
+    {
+      "name": "imagemagick",
```
Comment (TingPing) [inline io.github.guillaumechereau.Goxel.json]: This is only built to resize images or is it required for something else?

Comment (Alexander-Wilms) [reply io.github.guillaumechereau.Goxel.json]: Only for resizing the icon

**Comment by TingPing** [comment]
@guillaumechereau Do you have any interest in helping maintain this Flatpak? Since you already support Snap the selling points are largely similar as its a cross distro format.

File: io.github.guillaumechereau.Goxel.json (hunk 2 above)
Comment (TingPing) [reply io.github.guillaumechereau.Goxel.json]: Personally I think thats a bit unnecessary of a build dep and would rather just throw the made images in git and save a the world some wasted electricity.

**Comment by guillaumechereau** [comment]
I can certainly add all the required icons size in the git if that helps.

**Comment by guillaumechereau** [comment]
Thanks for the PR, I merged it.  I also added more icons sizes as: data/icons/<size>.png

**Comment by Alexander-Wilms** [comment]
Thanks.

@TingPing: Should I use the latest git commit then?

**Comment by guillaumechereau** [comment]
Yes I think that would be better, even though it doesn't match an actual release.

**Comment by TingPing** [comment]
Yea fine for now.

**Comment by Alexander-Wilms** [comment]
Okay, once https://github.com/guillaumechereau/goxel/pull/98 is merged, I'll update the manifest.

**Comment by Alexander-Wilms** [comment]
Could you trigger another build, please?

**Comment by nedrichards** [comment]
bot, build io.github.guillaumechereau.Goxel

**Comment by Alexander-Wilms** [comment]
I'm skipping i386 for now, so I think it's ready.

**Comment by nedrichards** [comment]
bot, build io.github.guillaumechereau.Goxel

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/io.github.guillaumechereau.Goxel

**Pull request closed without merging**